gradle-suppressible-error-prone/build.gradle

```diff
@@ -52,3 +52,8 @@ tasks.withType(JavaCompile) {
 gradleTestUtils {
     gradleVersions = ['8.14.3']
 }
+
+moduleJvmArgs {
```

Comment:
separate FLUP, but we should just add this to `gradle-test-utils` or whatever the plugin we have is called